core/src/main/java/hudson/util/ProcessTree.java

```diff
@@ -1980,5 +1980,6 @@ Object writeReplace() {
      * in case there's a fatal problem.
      *
      */
-    public static boolean enabled = !SystemProperties.getBoolean(ProcessTree.class.getName()+".disable");
+    public static boolean enabled = !SystemProperties.getBoolean("ProcessTreeKiller.disable")
```

Comment:
This should have the full package name https://www.jenkins.io/doc/book/managing/system-properties/#hudson-util-processtree-disable

Reply:
Ooops.